docker-compose.latest.yml

```diff
@@ -18,7 +20,7 @@ services:
     image: simplestakingcom/tezedge-explorer:latest
     hostname: explorer
     environment:
-      API: '[{"id":"localhost","name":"rust.dev.mainnet.localhost","http":"http://localhost:18732", "monitoring":"", "debugger":"", "ws":"ws://localhost:4927", "features":["MONITORING","MEMPOOL_ACTION","STORAGE_BLOCK"]}]'
+      API: '[{"id":"localhost","name":"rust.dev.mainnet.localhost","http":"http://localhost:18732", "monitoring":"", "debugger":"", "ws":"ws://localhost:4927", "features":["MONITORING","MEMPOOL_ACTION","STORAGE_BLOCK","RESOURCES"]}]'
```

Comment:
@tizoc 
does RESOURCES work without debugger?
because there is no debugger

Reply:
@bkontur what is missing is the monitoring page, which will show up as default and empty. What we want to be able to access is `resources -> storage`. Ideally we would have a flag that lets us enable just that.